src/nupic/utils/StringUtils.cpp

```diff
@@ -81,6 +81,24 @@ Real32 StringUtils::toReal32(const std::string& s, bool throwOnError, bool * fai
   return r;
 }
 
+Real64 StringUtils::toReal64(const std::string& s, bool throwOnError, bool * fail) {
```

Comment:
I couldn't find where this is used? 
- if unused, rm
- if at only one class, make it private of that class
- otherwise, ok

Reply:
> I couldn't find where this is used?

It isn't.  I added it because it was the only one of the data types that was not included.  It was in the .hpp but it had not been implemented.   It is very likely that none of the toXXX functions are used.

Reply:
> It is very likely that none of the toXXX functions are used. 

let's get killing LOC!

Reply:
moved to #378